crates/ruff_linter/src/rules/pylint/rules/too_many_branches.rs

```diff
@@ -16,54 +18,73 @@ use ruff_python_ast::identifier::Identifier;
 ///
 /// ## Example
 /// ```python
-/// def capital(country):
```

Comment:
Can we retain the old example as a simple first case? It can be in a separate code block e.g.

Instead of 

```
ex1
```

use

```
ex1
```

Instead of

```
ex2
```

use

```
ex2
```

Reply:
Done (I'm using the GitHub editor, so please squash these useless commits on merge)